src/components/themed/CryptoExchangeFlipInputWrapperComponent.js

```diff
@@ -165,7 +194,28 @@ const getStyles = cacheStyles((theme: Theme) => ({
     color: theme.primaryText,
     fontFamily: theme.fontFaceBold,
     fontSize: theme.rem(1.25)
+  },
+  balanceText: {
+    alignSelf: 'flex-start',
+    marginLeft: theme.rem(1),
+    marginBottom: theme.rem(0.5),
+    color: theme.secondaryText
   }
 }))
 
-export const CryptoExchangeFlipInputWrapper = withTheme(CryptoExchangeFlipInputWrapperComponent)
+export const CryptoExchangeFlipInputWrapper = connect<StateProps, {}, OwnProps>(
+  (state, ownProps) => {
+    const { displayCurrencyCode, displayDenomination } = ownProps.primaryCurrencyInfo
+    const balance = ownProps.guiWallet.nativeBalances[displayCurrencyCode]
+
+    if (balance != null) {
+      const cryptoAmountRaw: string = convertNativeToDenomination(displayDenomination.multiplier)(balance)
```

Comment:
refactor with Transaction List Header. Because they have the same balance display